2016/01/11 06:28:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_86_12_1411448128.jpg".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_86_12_1411448128.jpg
HTTP_REFERER=http://localhost:81/mypropertylk/list/property/type/buy
---
2016/01/11 06:28:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_87_1_1411448128.jpg".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_87_1_1411448128.jpg
HTTP_REFERER=http://localhost:81/mypropertylk/list/property/type/buy
---
2016/01/11 06:28:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_87_4_1411448129.jpg".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_87_4_1411448129.jpg
HTTP_REFERER=http://localhost:81/mypropertylk/list/property/type/buy
---
2016/01/11 06:28:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_87_3_1411448129.jpg".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_87_3_1411448129.jpg
HTTP_REFERER=http://localhost:81/mypropertylk/list/property/type/buy
---
